Dockerfile

```diff
@@ -1,25 +1,34 @@
-FROM golang:1.22
+# syntax = docker/dockerfile:1.2
+
+FROM golang:1.22 as builder
 
 ARG VERSION=dev
```

Comment:
Don't think this is being used by make as it is.

Reply:
Yes, legacy of when I was considering passing all the settings by enviroment variables

Reply:
OK [PR'ing](https://github.com/wtsi-npg/sqyrrl/pull/19) with line change